src/oge/data_pipeline.py

```diff
@@ -700,6 +700,11 @@ def main(args):
             year,
             args.skip_outputs,
         )
+        if not args.skip_outputs:
+            plant_attributes.assign(shaped_plant_id=pd.NA).to_csv(
```

Comment:
In fact, it is not tailored to the **plant_static_attributes** file because it will save the file in the **us_units** (or **metric_units*).

I guess this is reason why we don't use this function in Step 15 for the hourly results. See [here](https://github.com/singularity-energy/open-grid-emissions/blob/83bdce8f08d6120a3f66f60bdfd79cf9ec42d748/src/oge/data_pipeline.py#L564)

Reply:
Good point, we can leave as is. 
I'm wondering if in the future we should denormalize at least the annual plant data to include the plant name and some of the other attributes so that it is easier to browse.